FAQ.md

```diff
@@ -0,0 +1,71 @@
+# Frequently Asked Questions
+
+## What is the elevator pitch, why would I want to use kcp?
+
+kcp is a highly-multi-tenant Kubernetes control-plane, built for SaaS service-providers who want to offer a unified API-driven platform to many customers and dramatically reduce a) cost of operation of the platform and b) cost of onboarding of new services. It is made to scale to the order of magnitude of 10s of thousands of customers, offer a Kubernetes-like experience including execution of workloads, but with dramatically lower overhead per new customer.
+
+## What is a ....
+
+Check out our [terminology](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md) document and feel free to open an issue if something is not covered.
+
+
+## If kcp is a Kubernetes api server without pod-like APIs, how do resources like deployments get scheduled?
+
+kcp has a concept call [Syncer](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#syncer) which is installed on each [WorkloadClusters](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#workload-cluster). The [Syncer](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#syncer) negotaties, with kcp, a set of available APIs to make accessible in the workspace. This may include things like [Deployments](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/) or other resources you may explicitly configure the syncer to synchronize to kcp. Once these APIs are made available in your [Workspace](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#workspace) you may then create resources of that type. From there, the [Location and Placement](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#location) APIs help determine which [Location](https://github.com/kcp-dev/kcp/blob/main/docs/terminology.md#location) your deployable resource lands on.
+
+## Will KCP be able to pass the K8S conformance tests in [CNCF Conformance Suites](https://www.cncf.io/certification/software-conformance/)?
+
+For supported APIs, kcp is Kubernetes and will be compliant with conformance suites.  However, not all APIs are supported out of the box, on purpose.
```

Comment:
```suggestion
For supported APIs, kcp is Kubernetes and will be compliant with conformance suites. However, not all APIs are supported out of the box, on purpose.
```

Reply:
This response is kind of vague ... you need all the core APIs to pass conformance, so this needs more context

Reply:
Updated this to be more clear that we do not pass conformance out of the box since we don't support all APIs